src/main/java/frc/robot/Auto.java

```diff
@@ -114,10 +127,84 @@ public void runDefaultAutonomous() {
         }
     }
 
+    public void runBlueACubeLeaveCommunity() {
+        if (timer.get() < Blue_ACubeLeaveCommunity.getTotalTimeSeconds()) {
+            State desiredPose = Blue_ACubeLeaveCommunity.sample(timer.get());
+            ChassisSpeeds refChassisSpeeds = m_ramseteController.calculate(drivetrain.getPose(), desiredPose);
+
+            drivetrain.autoDrive(refChassisSpeeds.vxMetersPerSecond, refChassisSpeeds.omegaRadiansPerSecond);
+            System.out.println(Blue_ACubeLeaveCommunity.getTotalTimeSeconds()-timer.get());
+
+        } else {
+            drivetrain.autoDrive(0, 0);
+            System.out.println("FINISHED");
+        }
+    }
+
+    public void runBlueBlueACubeBalance() {
+        if (timer.get() < Red_ACubeBalance.getTotalTimeSeconds()) {
+            State desiredPose = Red_ACubeBalance.sample(timer.get());
+            ChassisSpeeds refChassisSpeeds = m_ramseteController.calculate(drivetrain.getPose(), desiredPose);
+
+            drivetrain.autoDrive(refChassisSpeeds.vxMetersPerSecond, refChassisSpeeds.omegaRadiansPerSecond);
+            System.out.println(Red_ACubeBalance.getTotalTimeSeconds()-timer.get());
+
+        } else {
+            drivetrain.autoDrive(0, 0);
+            System.out.println("FINISHED");
+            finishedTrajectory = true;
```

Comment:
this is the only one of these similar methods to set finishedTrajectory.  is it used?  should all the methods do it?